src/pages/Restaurants/restaurantsList.tsx

```diff
@@ -1,39 +1,240 @@
+      popular: false,
+      open: true,
+      type: "restaraunt",
+      title: "Kab Kem",
+      image: "../images/kab-kem.png",
+      body: "Yariv Malili",
+      alt: "Kab Kem",
+    },
+    {
+      id: 3,
+      new: false,
+      popular: false,
+      open: false,
+      type: "restaraunt",
+      title: "Messa",
+      image: "../images/messa.png",
+      body: "Aviv Moshe",
+      alt: "Messa",
+    },
+    {
+      id: 4,
+      new: true,
+      popular: false,
+      open: false,
+      type: "restaraunt",
+      title: "Nithan Thai",
+      image: "../images/nithan-thai.png",
+      body: "Shahaf Shabtay",
+      alt: "Nithan Thai",
+    },
+    {
+      id: 5,
+      new: false,
+      popular: false,
+      open: true,
+      type: "restaraunt",
+      title: "Tiger Lilly",
+      image: "../images/tiger-lilly.png",
+      body: "Yanir Green",
+      alt: "Tiger Lilly",
+    },
+    {
+      id: 6,
+      new: true,
+      popular: true,
+      open: true,
+      type: "restaraunt",
+      title: "Ya Pan",
+      image: "../images/japanese.png",
+      body: "Yuval Ben Neriah",
+      alt: "Ya Pan",
+    },
+    {
+      id: 7,
+      new: true,
+      popular: false,
+      open: false,
+      type: "restaraunt",
+      title: "Lumina",
+      image: "../images/lumina.png",
+      body: "Meir Adoni",
+      alt: "Lumina",
+    },
+  ];
+
+
+
+  const [restList, setRestList] = useState(restarauntList);
+//   const [status, setStatus] = useState('all');
+  
+
+//   const filterHandler = (value) => {
+//       switch (status) {
+
+//           case "all":
+//             setFilteredRestaraunts(restarauntList.filter(card => card) )   
+//                   break;
     
-    ]
-
-
-    return (
-        <div>
-            {list.map(card => {
-                return (
-                    <Card key={card.id} title={card.title} price={card.price} body={card.body} img={card.image} alt={card.alt}/>
-                )
-            })}
-        </div>
-    )
+//           case "new":
+//             setFilteredRestaraunts(restarauntList.filter(card => card.new))  
+//           break;    
+
+//           case "popular":
+//             setFilteredRestaraunts(restarauntList.filter(card => card.popular))    
+//           break;
+          
+//           case "open":
+//             setFilteredRestaraunts(restarauntList.filter(card => card.open))   
+//           break;      
+//   }
+const filterHandler = (e:any) => {
+    if (e.target.value === 'all') {
+       let filtered =  setRestList(restarauntList.filter(card => card) )   
+    }
+    if(e.target.value === 'new') {
+        console.log(e.target.value)
+        let filtered =  restarauntList.filter(card => card.new) 
+        setRestList(filtered)
+    }
+    if(e.target.value === 'open') {
+        setRestList(restarauntList.filter(card => card.open))  
+    }
+    if(e.target.value === 'popular') {
+        setRestList(restarauntList.filter(card => card.popular))    
+    }
```

Comment:
This is a classic case for switch statement, just like you did in the comment :)
Good job on the filtering, nice implementation

Reply:
It's under construction because now the data is from firebase. 
Thanks for the tip!